usaspending_api/search/tests/unit/filters/naics.py

```diff
@@ -0,0 +1,37 @@
+from usaspending_api.search.elasticsearch.filters.naics import NaicsCodes
+
+
+def test_primative_naics_filter():
+    assert NaicsCodes._query_string(["11"], []) == "(11*)"
```

Comment:
Small details but might be good to change the test cases to call out the parameter since it is just two lists next to each other. For example, `assert NaicsCodes._query_string(requires=["11"], exclude=[]) == "(11*)"`. Will help anyone who goes behind you on these test cases.